src/state/nftMarket/helpers.ts

```diff
@@ -254,101 +351,29 @@ export const fetchWalletTokenIdsForCollections = async (
   return walletNfts.flat()
 }
 
-/**
- * Get the most recently listed NFTs
- * @param first Number of nfts to retrieve
- * @returns NftTokenSg[]
- */
-export const getLatestListedNfts = async (first: number): Promise<NftTokenSg[]> => {
```

Comment:
Lots of lines in diff here are just moving functions up or down (to group API, Subgraph and other helpers).
Most real changes are just types and some other minor things